In [1]:
from scipy import stats
from scipy.stats import norm

In [5]:
Pop mean =270 days
Sample mean = 260 days
Sample SD = 90 days
Sample n = 18 bulbs
df = n-1 = 17

SyntaxError: invalid syntax (<ipython-input-5-e80929aa649f>, line 1)

In [4]:
# find t-scores at x=260; t=(s_mean-P_mean)/(s_SD/sqrt(n))
t=(260-270)/(90/18**0.5)
t

-0.4714045207910317

In [6]:
# Find P(X>=260) for null hypothesis

In [7]:
# p_value=1-stats.t.cdf(abs(t_scores),df=n-1)... Using cdf function
p_value=1-stats.t.cdf(abs(-0.4714),df=17)
p_value

0.32167411684460556

In [8]:
#  OR p_value=stats.t.sf(abs(t_score),df=n-1)... Using sf function
p_value=stats.t.sf(abs(-0.4714),df=17)
p_value

0.32167411684460556

Probability that 18 randomly selected bulbs would have an average life of no more than 260 days is 32.17%

Assuming significance value α = 0.05 (Standard Value)
(If p_value < α ; Reject Ho and accept Ha or vice-versa)
Thus, as p-value > α ; Accept Ho i.e. The CEO claims are false and the avg life of bulb > 260 days